### DISCUSSION TITLE
"由于目标计算机积极拒绝，无法连接"？

### DISCUSSION THREAD

**Discussion by xiyuchengxu** [mikumifa/biliTickerBuy#143] [提问专区]
app.log是空的。![Screenshot_2024-06-20-00-33-16-235_com.microsoft.rdc.androidx.jpg](https://github.com/mikumifa/biliTickerBuy/assets/89074654/7f583715-fcd6-4728-8176-ad93c6cf3d64)

**Comment by mikumifa** [comment]
先重启一下电脑，看看你能不能连上网
